tests/credentials_tests.rb

```diff
@@ -1,6 +1,31 @@
 # frozen_string_literal: true
 
 Shindo.tests('AWS | credentials', ['aws']) do
+  unchecked_constants = [:CredentialFetcher, :VERSION, :Errors, :Mock, :ServiceMapper, :Federation, :STS, :SignatureV4, :EFS]
```

Comment:
The reason for the above constructs' exclusion from the test is below.

1. CredentialFetcher, VERSION, Errors, Mock, ServiceMapper are not inherited Fog::Service. 

2. Federation, SignatureV4 are not `requires` and `recognizes` defined.

3. STS is `aws_access_key_id` and `aws_secret_access_key` parameters are defined at `recognizes` instead of `requires`.

4. EFS is not defined `aws_session_token` on `recognizes`.